doc/dune

```diff
@@ -0,0 +1,6 @@
+
+(mdx 
+ (packages current_examples)
+ (preludes prelude.ml)
```

Comment:
I suggest getting rid of this predule. It contains things the user actually needs to type, so hiding it isn't helpful.